src/js/jsx/sections/layers/LayerFace.jsx

```diff
@@ -408,10 +418,12 @@ define(function (require, exports, module) {
                 this._isDragEventTarget = false;
             }
             
-            this.setState({
-                isDragging: false,
-                dragStyle: null
-            });
+            if (this.isMounted()) {
```

Comment:
Using `isMounted` is apparently considered an anti-pattern by the React team, and I think the method is slated for deprecation. @volfied recently went through the project and removed all such instances. He can give you some suggestions for the best way to deal with this.